src/tools/perf/ucp_tests.cc

```diff
@@ -44,12 +44,13 @@ class ucp_perf_test_runner {
         iov_length_it = 0;
         for (iov_it = 0; iov_it < iovcnt; ++iov_it) {
             iov[iov_it].buffer = (char *)buffer + iov_length_it;
-            iov[iov_it].length = m_perf.params.msg_size_list[iov_it];
+            iov[iov_it].dt     = ucp_dt_make_contig(m_perf.params.msg_size_list[iov_it]);
```

Comment:
i think it's better to not put any iov implementation here, since it's very incomplete (e.g only contig types are supported); just rename length to count and if "dt" is not ucp_dt_make_contig(1), return INVALID_PARAM.